rebench/interop/validation_log_adapter.py

```diff
@@ -61,7 +61,7 @@ def parse_data(self, data, run_id):
                 if m.group(5) == "u":
                     time /= 1000
                 criterion = (m.group(2) or 'total').strip()
-                successMeasure = Measurement(bool(m.group(5)), 'bool', run_id, 'Success')
+                successMeasure = Measurement(bool(m.group(6) == "true"), 'bool', run_id, 'Success')
```

Comment:
now with the `==` giving you a bool, you don't need the `bool(..)` around it I suppose?